fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -266,7 +266,11 @@ abstract class BaseBrowserFragment :
         _binding = FragmentBrowserBinding.inflate(inflater, container, false)
 
         val activity = activity as HomeActivity
-        activity.themeManager.applyStatusBarTheme(activity)
+        // ExternalAppBrowserActivity handles it's own theming as it can be customized.
+        if (activity !is ExternalAppBrowserActivity) {
+            activity.themeManager.applyStatusBarTheme(activity)
+        }
```

Comment:
Can we have private custom tabs? Would this then prevent the private theming of them?

Reply:
So afaikt, this would be [set here](https://github.com/mozilla-mobile/firefox-android/blob/8832039332e80981ae980d4b84c5c3731ea6df94/fenix/app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt#L44) and would not be overridden because [of this gate](https://github.com/mozilla-mobile/firefox-android/blob/8832039332e80981ae980d4b84c5c3731ea6df94/fenix/app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt#L69)